src/components/SiteTitle.js

```diff
@@ -1,15 +1,10 @@
 import React from 'react';
 
 const SiteTitle = () => (
-  <div className="text-center">
+  <div className="text-center hero-title">
     <a className="no-underline inline-block py-12 md:py-8" href="/">
-      <h1 className="text-dark-pink text-6xl md:text-7xl">
-        Hacktober
-        <span className="text-light-pink text-3xl md:text-5xl uppercase block">
-          fest
-        </span>
-      </h1>
-      <small className="block text-light-blue light-mode:text-dark-blue uppercase block">
+      <img src="https://hacktoberfest.digitalocean.com/assets/HF-full-logo-b05d5eb32b3f3ecc9b2240526104cf4da3187b8b61963dd9042fdc2536e4a76c.svg" />
```

Comment:
Although I agree this looks really good, I don't think we should do this. Although I _think_ we're ok to use the logo, I think we'd have to accompany it with the `Presented by...` logos too (see https://www.figma.com/proto/oc9p0mS6A0cAOOhipLT9aV/DODX-959-Hacktoberfest-2020-Brand-Guidelines?node-id=2%3A4401&viewport=-36%2C134%2C0.37746772170066833&scaling=min-zoom) even if we did that I think it makes sense to keep a separation with the logo, I've tried to get as close as possible without ripping directly.

Reply:
Got it, still keep the old logo.